ArduCopter/Copter.cpp

```diff
@@ -889,7 +889,7 @@ void Copter::update_super_simple_bearing(bool force_update)
     }
 
     super_simple_last_bearing = bearing;
-    const float angle_rad = radians((super_simple_last_bearing+18000)/100);
+    const float angle_rad = radians((float)((super_simple_last_bearing+18000)/100));
```

Comment:
I think * 0.01 would be better and fix an actual rounding bug

Reply:
Done, thanks.